lightning/src/ln/channelmanager.rs

```diff
@@ -10367,14 +10375,23 @@ where
 				};
 
 				match result {
-					Ok(()) => ResponseInstruction::NoResponse,
-					Err(e) => match responder {
-						Some(responder) => responder.respond(OffersMessage::InvoiceError(e)),
+					Err(err) => match responder {
+						Some(responder) => {
+							abandon_if_payment(context);
+							responder.respond(OffersMessage::InvoiceError(err))
+						}
 						None => {
-							log_trace!(self.logger, "No reply path for sending invoice error: {:?}", e);
-							ResponseInstruction::NoResponse
-						},
-					},
+							abandon_if_payment(context);
+							log_trace!(
+								self.logger,
+								"A error response was generated, but there is no reply_path specified \
```

Comment:
s/A/An

Reply:
Done! [528dc64](https://github.com/lightningdevkit/rust-lightning/commit/528dc644e5b995c10f96fe36fb803bc6433580d7)
Must have missed it while I fixed the other log message 😅
Thanks for pointing it out!